linkerd/app/core/src/svc.rs

```diff
@@ -51,6 +51,10 @@ impl<L> Layers<L> {
         self.push_pending().push(buffer::layer(bound, d))
     }
 
+    pub fn push_per_make<T: Clone>(self, layer: T) -> Layers<Pair<L, stack::per_make::Layer<T>>> {
```

Comment:
TIOLI: It might be worth adding a comment to explain the difference between `push` and `push_per_make`?

Reply:
This will change more. We'll document it better as it stabilizes.